CHANGELOG.md

```diff
@@ -23,6 +23,8 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/virtual-users/compare/release-3.7.0...master
 
+- Add SPI for users generation
```

Comment:
Let's add the `### Added` section

Reply:
PS. A public ticket would be nice, because we can link to it and discuss the feature.